tests/integration/seed/seedsVerification.test.js

```diff
@@ -161,19 +187,208 @@ describe('SEEDS VERIFICATION', () => {
+            });
+
+          expect(someAccessRulesAreInDuplicate).toBeFalsy();
+        });
+
+        it('should pass if every subprogram exists', () => {
+          const subProgramsExist = courseList.map(course => course.subProgram).every(subProgram => !!subProgram);
+          expect(subProgramsExist).toBeTruthy();
+        });
+
+        it('should pass if every company exists and is not in duplicate', async () => {
+          const someCompaniesDontExist = courseList
+            .filter(c => c.format === BLENDED)
+            .some(c => c.companies.some(company => company === 'company not found'));
+
+          expect(someCompaniesDontExist).toBeFalsy();
+
+          const someCompaniesAreInDuplicate = courseList
+            .filter(course => get(course, 'companies.length'))
+            .some((course) => {
+              const companiesWithoutDuplicates = [...new Set(course.companies.map(c => c._id.toHexString()))];
+
+              return course.companies.length !== companiesWithoutDuplicates.length;
+            });
+
+          expect(someCompaniesAreInDuplicate).toBeFalsy();
+        });
+
+        it('should pass if intra courses have one and only one company', () => {
+          const everyIntraCourseHasCompany = courseList
+            .filter(course => course.type === INTRA)
+            .every(course => course.companies.length === 1);
+
+          expect(everyIntraCourseHasCompany).toBeTruthy();
+        });
+
+        it('should pass if no e-learning course has companies field', () => {
+          const someELearningCourseHasCompanies = courseList
+            .filter(course => course.format === STRICTLY_E_LEARNING)
+            .some(course => has(course, 'companies'));
+          expect(someELearningCourseHasCompanies).toBeFalsy();
+        });
+
+        it('should pass if every blended course is intra ou inter_b2b', () => {
+          const everyBlendedCourseHasGoodType = courseList
+            .filter(course => course.format === BLENDED)
+            .every(course => [INTRA, INTER_B2B].includes(course.type));
+
+          expect(everyBlendedCourseHasGoodType).toBeTruthy();
+        });
+
+        it('should pass if every strictly e-learning course is inter_b2c', () => {
+          const everyELearningCourseHasGoodType = courseList
+            .filter(course => course.format === STRICTLY_E_LEARNING)
+            .every(course => course.type === INTER_B2C);
+
+          expect(everyELearningCourseHasGoodType).toBeTruthy();
+        });
+
+        it('should pass if trainer is never in trainees list', () => {
+          const isTrainerIncludedInTrainees = courseList
+            .filter(c => has(c, 'trainer'))
+            .some(c => UtilsHelper.doesArrayIncludeId(c.trainees.map(t => t._id), c.trainer._id));
+          expect(isTrainerIncludedInTrainees).toBeFalsy();
+        });
+
+        it('should pass if every trainee exists and is not in duplicate', () => {
```

Comment:
tu es sur de ton "in duplicate" ? 😅 ce n'est pas plutot "duplicated" ?

Reply:
l'adjectif c'est duplicate, mais le 'in' est de trop